assets/monitoring/grafana/v1alpha1/dashboards/saas/overview.json

```diff
@@ -0,0 +1,4680 @@
+{
```

Comment:
@amnonh this one is not updated as I haven't found a source for it in 4.4.5 - where can I find it?
(also this will break with grafana 10.0 for the 4.5 bump)